### ISSUE TITLE
smbcontrol triggers AVC in SELinux enforcing mode

### ISSUE THREAD

**Issue by flo-renaud** [fedora-selinux/selinux-policy#950]
The command smbcontrol fails in enforcing mode and triggers an AVC.

In order to reproduce:
```
# dnf install -y freeipa-server-trust-ad
# rpm -qa samba selinux-policy
selinux-policy-35.5-1.fc35.noarch
samba-4.15.2-3.fc35.x86_64
# smbcontrol all debug 100
ERROR: Could not determine network interfaces, you must use a interfaces config line
# ausearch -m AVC
[...]
type=AVC msg=audit(1637670284.741:727): avc:  denied  { create } for  pid=6159 comm="smbcontrol" scontext=unconfined_u:unconfined_r:smbcontrol_t:s0-s0:c0.c1023 tcontext=unconfined_u:unconfined_r:smbcontrol_t:s0-s0:c0.c1023 tclass=netlink_route_socket permissive=0
```

**Comment by flo-renaud** [comment]
Also tracked against samba at https://bugzilla.samba.org/show_bug.cgi?id=14917

**Comment by flo-renaud** [comment]
And tracked against fedora 35: https://bugzilla.redhat.com/show_bug.cgi?id=2025931

**Comment by zpytela** [comment]
The selinux-policy BZ should allow the use of sockets which were not required previously.

**Comment by zpytela** [comment]
Fix for BZ 2025931 is available in rawhide and F35 updates-testing.
